lib/src/rules/avoid_returning_null_for_future.dart

```diff
@@ -63,17 +63,17 @@ class _Visitor extends SimpleAstVisitor<void> {
     if (parent == null) return;
 
     DartType? returnType;
```

Comment:
The `declaredElement` must be always set (to `ExecutableElement`) in resolved units, so the return type is never `null`.

Reply:
Are you suggesting `!`s below?:

    returnType = parent.declaredElement!.returnType;